indexer/services/comlink/__tests__/controllers/api/v4/transfers-controller.test.ts

```diff
@@ -27,6 +27,13 @@ import {
 } from '@dydxprotocol-indexer/postgres/build/__tests__/helpers/constants';
 import Big from 'big.js';
 
+const defaultWallet = {
```

Comment:
nit: why not use testConstants.defaultWallet imported from postgres?

Reply:
good point